dt-contacts/base-setup.php

```diff
@@ -61,6 +61,8 @@ public function dt_get_post_type_settings( $settings, $post_type ){
         if ( $post_type === $this->post_type ){
             $settings['label_singular'] = __( 'Contact', 'disciple_tools' );
             $settings['label_plural'] = __( 'Contacts', 'disciple_tools' );
+            $settings['status_field'] = "overall_status";
+            $settings['archived_status_key'] = "closed";
```

Comment:
@squigglybob 
can we do something like this so we can expand it more later if needed?
```
status_field => [
field_key=> "overall_status"
archived_key=> "closed"
]
```

Reply:
alternately we would add a flag to the existing status field

Reply:
good idea. what sort of things do you think it might expand into? Would it always be to do with the status field? If not maybe we could give it a more generic name instead of status_field...